moodler/course.py

```diff
@@ -0,0 +1,30 @@
+from collections import defaultdict
+
+from moodler.moodle_api import call_moodle_api
+
+
+def get_course_exercises(courseid: int):
+    """
+    Retrieve IDs of LTI exercises and assignments mapped to their name
```

Comment:
the function name doesn't follow the docstring. It provides too specific behavior. It also doesn't return all exercises that Moodle returns, but a filtered subset AND it's grouped by `modname`
What if the user needs a full list and all information returned AND will change the function? It will break integrations.
Shouldn't filtering and grouping be done in places where the information is used?

Reply:
Removed this function from moodler and moved it to dart training script instead. The `core_course_get_contents` moodle api call has already been implemented in `moodler.sections`.